op-batcher/batcher/config.go

```diff
@@ -96,11 +96,8 @@ type CLIConfig struct {
 	// ActiveSequencerCheckDuration is the duration between checks to determine the active sequencer endpoint.
 	ActiveSequencerCheckDuration time.Duration
 
-	// ThrottleInterval is the interval between notifying the block builder of the latest DA throttling state, or 0 to
+	// ThrottleThreshold is the number of pending bytes beyond which the batcher will start throttling future bytes. Set to 0 to
 	// disable notifications entirely (only recommended for testing).
```

Comment:
```suggestion
	// ThrottleThreshold is the number of pending bytes beyond which the batcher will start throttling future bytes. Set to 0 to
	// disable sequencer throttling entirely (only recommended for testing).
```
Clearer that throttling isn't an internal process, but works on the sequencer.

You may also need to update the `flags.ThrottleThresholdFlag` description.